src/AAT.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @description PocketAAT entry point.
+ */
+
+import { sha3_256 } from 'js-sha3';
+import ed25519 = require('ed25519');
+
+export class PocketAAT {
+  public readonly version: string;
+  public readonly clientPublicKey: string;
+  public readonly applicationPublicKey: string;
+  private applicationSignature: string;
+
+  constructor(version: string, clientPublicKey: string, applicationPublicKey: string) {
```

Comment:
Should we have version be an enum or constant, as these versions are always going to be derived from from the AAT specification?

Reply:
added a constant for a default_version value